unison-src/transcripts-manual/rewrites.output.md

```diff
@@ -306,21 +306,17 @@ Instead, it should be an unbound free variable, which doesn't typecheck:
 
   Loading changes detected in scratch.u.
 
-  I couldn't find any definitions matching the name bar21 inside the namespace .
+  I couldn't figure out what bar21 refers to here:
   
      19 |   bar21
   
+  Its type should conform to _ .
```

Comment:
What's up with the type `_` here? Doesn't seem right 🤔

Reply:
Hmm, used to be "there are no constraints on its type". I must have botched the pattern match on the type here.